eta/core/geometry.py

```diff
@@ -55,6 +55,27 @@ def __eq__(self, bbox):
             self.bottom_right == bbox.bottom_right
         )
 
+    @property
+    def top_right(self):
+        '''Returns a top right RelativePoint'''
+        return RelativePoint(self.bottom_right.x, self.top_left.y)
+
+    @property
+    def bottom_left(self):
+        '''Returns a bottom left RelativePoint'''
+        return RelativePoint(self.top_left.x, self.bottom_right.y)
+
+    @property
+    def corners(self):
+        '''Returns all four RelativePoint corners in clockwise order starting with
+        the top left corner.
+        
+        Returns:
+            a 4-tuple of RelativePoints
```

Comment:
small suggestion: could write this as `(top_left, top_right, bottom_right, bottom_left) tuple of RelativePoints` to make it easier on the reader of the documentation only

Reply:
Done!